### DISCUSSION TITLE
categoryIdentifier not received

### DISCUSSION THREAD

**Discussion by raonikhil** [ctwillie/expo-server-sdk-php#22] [Q&A]
I am sending this data via the SDK. I receive everything except for the categoryId as categoryIdentifier. Is there anything that I am doing wrong?

```
$messages = [
     'title'      => $notification['title'],
     'body'       => $notification['body'],
     'to'         => $notification['device_token'],
     'data'       => json_decode($notification['json_data'], true),
     'categoryId' => $notification['category_identifier'],
];
```

**Comment by ctwillie** [comment]
Currently, only properties on the `ExpoMessage` class are sent as the notification payload. Any key not found as a property in the class is ignored. Feel free to create a PR if you find something is missing. ExpoMessage class [here](https://github.com/ctwillie/expo-server-sdk-php/blob/master/src/ExpoMessage.php).
